src/words.jl

```diff
@@ -74,3 +80,18 @@ Base.@propagate_inbounds function Base.setindex!(w::Word, v::Integer, n::Integer
     @assert v > 0 "All entries of a Word must be positive integers"
     return @inbounds w.ptrs[n] = v
 end
+
+"""
+    lcp(u::Word, v::Word)
+Returns the length of longest common prefix of two words (and simultaneously
+the index at which the prefix ends).
+"""
+function lcp(u::Word, v::Word)
+    n=0
+    for i in zip(u,v)
+        if i[1] == i[2]
+            n += 1
+        end
```

Comment:
if we're using both elts in zip, then it's better to
```julia
for (lu, lv) in zip(u,v)
    lu != lv && break
    n += 1
end
```
indexing to `i`has small penalty for bounds check, or maybe used to have?